style.css

```diff
@@ -104,9 +105,13 @@ main {
 /* Header */
 
 header.box {
-  position: relative;
+  position: fixed;
+  width: 100%;
+  background-color: #fff;
```

Comment:
Is this really needed? I removed it and it looked the same to me.
Also, for info, we have a hidden dark theme. If you click on the title in the header, the site switches to dark. So, if you need to make something white in the future, then remember to also make it black/dark when the dark theme is ON (the class `.dark` is applied to the body when this happens).